frontend/src/metabase/query_builder/components/filters/modals/InlineValuePicker/InlineValuePicker.styled.ts

```diff
@@ -26,15 +26,15 @@ export const ValuesPickerContainer = styled.div`
 
 export const BetweenContainer = styled.div`
   display: flex;
+  height: 53px;
   width: 100%;
   align-items: center;
-  justify-content: space-between;
 `;
```

Comment:
with a wider modal, spreading the between inputs out looked bad
![Screen Shot 2022-07-08 at 11 12 45 AM](https://user-images.githubusercontent.com/30528226/178039204-1b649826-d8b2-4667-9db1-83049069339d.png)